bin/templates/project/__PROJECT_NAME__/config.xml

```diff
@@ -53,7 +53,7 @@
     <preference name="DisallowOverscroll" value="false" />
     <preference name="EnableViewportScale" value="false" />
     <preference name="KeyboardDisplayRequiresUserAction" value="true" />
-    <preference name="MediaPlaybackRequiresUserAction" value="false" />
+    <preference name="MediaTypesRequiringUserActionForPlayback" value="false" />
```

Comment:
This is for `mediaTypesRequiringUserActionForPlayback` which (already) included in other PR https://github.com/apache/cordova-ios/pull/784.
Or the `AllowsAirPlayForMediaPlayback` property here?

Reply:
Read the last commit. First commit is from the last Pr. This was built off of the last Pr and should be merged first.